tests/test_session.py

```diff
@@ -274,6 +274,7 @@ def test_get_fluent_mode(new_mesh_session):
 
 @pytest.mark.dev
 @pytest.mark.fluent_232
+@pytest.mark.skip("Failing in github")
```

Comment:
This test often fails in github probably due to the issue demonstrated in https://github.com/pyansys/pyfluent/pull/1302